ro-base.obo

```diff
@@ -1913,6 +1920,7 @@ subset: ro-eco
 property_value: IAO:0000114 IAO:0000125
 property_value: IAO:0000117 https://orcid.org/0000-0002-6601-2165
 property_value: IAO:0000118 "produces" xsd:string
+property_value: seeAlso Has:output
```

Comment:
?

Reply:
Fixed in the commit [c0ecc1](https://github.com/oborel/obo-relations/pull/728/commits/c0ecc1df96a957bd979bc98214b85b855e6818ad)